lib/java/src/main/java/org/apache/thrift/TException.java

```diff
@@ -20,7 +20,7 @@
 package org.apache.thrift;
 
 /** Generic exception class for Thrift. */
-public class TException extends Exception {
+public class TException extends RuntimeException {
```

Comment:
this is a breaking change

Reply:
It's due to Apache HttpComponent 5, good usage is to use http call code in lambda.

Reply:
can you elaborate on that? why changing to runtime exception is a must due to a library change?

Reply:
I have coded http call in inline method, now it's better